src/core/utils.hpp

```diff
@@ -31,7 +31,7 @@ using Matrix = std::array<std::array<T, COLS>, ROWS>;
 /// Polyfill ///
 template<typename T>
 constexpr T abs(T num) {
-#ifdef IS_APPLE
+#ifdef __APPLE__
 	// Apple Clang is a "good" compiler...
 	if (num < 0) return -num;
```

Comment:
I wonder if this workaround is still needed with newer versions of clang...maybe the ifdef can be removed entirely?

Reply:
I don't know, what should it affect on the game mechanics?

Reply:
IIRC the apple compiler didn't support constexpr std::abs when this was written, but I believe now it does, so the entire function could just return std::abs(num) on all platforms. Could you try removing the ifdef and see if it compiles?